sdk/core/azure-core-http-okhttp/src/main/java/com/azure/core/http/okhttp/OkHttpAsyncClientProvider.java

```diff
@@ -10,7 +10,7 @@
 /**
  * An {@link HttpClientProvider} that provides an implementation of HttpClient based on OkHttp.
  */
-public final class OkHttpClientProvider implements HttpClientProvider {
+public final class OkHttpAsyncClientProvider implements HttpClientProvider {
```

Comment:
This is a breaking change. The package and the class both are public.

Reply:
These were moved into public API earlier this month and haven't been released yet